### PR TITLE
prevent crashing if fingerprint is not supported

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Settings crashed on < M

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/activity/Preferences.java [hunk 1]
```diff
@@ -189,41 +189,49 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
...
-                                        .show();
-                                fPrint.setChecked(false);
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                if (FingerprintActivity.isFingerprintCapable(MainApp.getAppContext()) && fPrintEnabled) {
+                    fPrint.setOnPreferenceChangeListener(new OnPreferenceChangeListener() {
+                        @Override
+                        public boolean onPreferenceChange(Preference preference, Object newValue) {
+                            Boolean incoming = (Boolean) newValue;
+
+                            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/activity/Preferences.java]: Line 192 already checked for build version :)

**Comment by AndyScherzinger** [comment]
👍 fine with me, just found a duplicate check

File: src/main/java/com/owncloud/android/ui/activity/Preferences.java (hunk 1 above)
Comment (mario) [inline src/main/java/com/owncloud/android/ui/activity/Preferences.java]: Android studio would complain otherwise xD

**Comment by mario** [comment]
👍

**Pull request merged**